test/app/features/help_center/data/datasources/guardian_datasources_fetch_test.dart

```diff
@@ -78,31 +81,31 @@ void main() {
 
       group('fetch()', () {
         test(
-          'should perform a GET with X-API-Key',
+          'perform a GET with X-API-Key',
           () async {
             // arrange
             const endPointPath = '/me/guardioes';
             final headers = await _setUpHttpHeader();
-            final request = _setuHttpRequest(endPointPath, {});
+            final request = _setUpHttpRequest(endPointPath, {});
             _setUpMockGetHttpClientSuccess200(bodyContent);
             // act
             await dataSource.fetch();
             // assert
-            verify(apiClient.get(request, headers: headers));
+            verify(() => apiClient.get(request, headers: headers)).called(1);
           },
         );
         test(
-          'should get a valid GuardianSession for a successful request',
+          'get a valid GuardianSession for a successful request',
           () async {
             // arrange
             _setUpMockGetHttpClientSuccess200(bodyContent);
             final jsonData =
                 await JsonUtil.getJson(from: 'help_center/guardian_list.json');
-            final expected = GuardianSessionModel.fromJson(jsonData);
+            final actual = GuardianSessionModel.fromJson(jsonData);
             // act
             final received = await dataSource.fetch();
             // assert
-            expect(expected, received);
+            expect(actual, received);
```

Comment:
Os argumentos do expect estão invertidos, da maneira que esta caso o teste falhe a mensagem de erro será confusa, mostrando que o valor retornado como se fosse o correto (esperado).

```suggestion
            final expected = GuardianSessionModel.fromJson(jsonData);
            // act
            final received = await dataSource.fetch();
            // assert
            expect(received, expected);
```